src/utils/searchUtils.ts

```diff
@@ -21,9 +21,10 @@ import type {
 function getSortQuery(sortBy = "", order = ""): string {
   const reset = sortBy === "relevance"
   let sortQuery = ""
+  const sortDirectionQuery = order === "" ? "" : `&sort_direction=${order}`
 
   if (sortBy?.length && !reset) {
-    sortQuery = `&sort=${sortBy}&sort_direction=${order}`
+    sortQuery = `&sort=${sortBy}${sortDirectionQuery}`
```

Comment:
Should `sort=${...}` also be dealt the same way?

Reply:
line 27 is only reached if `sortBy?.length` is true. are you concerned about the logic being parallel or sort being empty?

Reply:
you're right. i just ignored that check initially so ignore the comment.